### ISSUE TITLE
[Selectable tile] The checkmark bx--tile__checkmark is barely visible for high contrast mode users because it does not maintain a 3:1 contrast ratio

### ISSUE THREAD

**Issue by cheer021** [carbon-design-system/carbon#8191]
## Environment

> Operating system

Windows 

> Browser

FF (should be across browsers) 

> Automated testing tool and ruleset

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?
latest 

> What did you expect to happen?
maintain a 3:1 contrast ratio

> What happened instead?
does not maintain a 3:1 contrast ratio

> What WCAG 2.1 checkpoint does the issue violate?

maintain a 3:1 contrast ratio

## Steps to reproduce the issue

1. On your Windows machine, turn on High Contrast # 2 theme
2. Browsing on Firefox ESR
3. single-click on a tile to select it
4. Look at the checkmark that shows

## Additional information

- reported by our QCs: 

All icons in Cognos Analytics should adhere to the IBM Accessibility Checklist's minimum contrast ratio

![Screen Shot 2021-03-24 at 6 41 51 PM](https://user-images.githubusercontent.com/2319230/112569413-baecc280-8da9-11eb-94bd-073d4ba7d467.png)

**Comment by tw15egan** [comment]
Should be resolved via https://github.com/carbon-design-system/carbon/pull/8489
